pkg/controller/clientutil/indexers_test.go

```diff
@@ -0,0 +1,22 @@
+package clientutil_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/controller/clientutil"
+)
+
+func TestClusterNameIndexerValid(t *testing.T) {
```

Comment:
test for the panic? I think gomega has something to make it easier to test

Reply:
I initially didn't find it but I did now, changing